source/docs/advanced/graphql/remote-schemas.md

```diff
@@ -182,9 +182,29 @@ module.exports = {
 
 ## Caveats
 
+### Server restart needed
+
 Please note that **remote schema changes will only appear in your schema after a server restart**.
 The remote schema definition is fetched during the server bootstrap process.
 
+### Query root type must be provided
+
+The remote schema (after transforms applied) **MUST** contain at least a `Query` root type.
```

Comment:
I would mention that this is the case when you only define a mutation or that you filter all the Queries from your remote schema.